test/performance/kueue/config.yaml

```diff
@@ -0,0 +1,175 @@
+
+{{$smallJobsPerNamespace := DivideInt $smallJobs $namespaces}}
+{{$mediumJobsPerNamespace := DivideInt $mediumJobs $namespaces}}
+{{$largeJobsPerNamespace := DivideInt $largeJobs $namespaces}}
+
+{{$smallJobSize := 5}}
+{{$mediumJobSize := 20}}
+{{$largeJobSize := 100}}
+
+{{$jobRunningTime := DefaultParam .CL2_JOB_RUNNING_TIME "30s"}}
+
+{{$clusterQueue := "default-cluster-queue"}}
+{{$localQueue := "local-queue"}}
+
+{{$testTimeout := DefaultParam .CL2_TEST_TIMEOUT "5m"}}
+
+{{$namespacePrefix := "queue-test"}}
+
+name: batch
+
+namespace:
+  number: {{$namespaces}}
+  prefix: {{$namespacePrefix}}
+
+tuningSets:
+- name: UniformQPS
+  qpsLoad:
+    qps: {{$LOAD_TEST_THROUGHPUT}}
+
+steps:
+- name: Start measurements
+  measurements:
+  - Identifier: Timer
+    Method: Timer
+    Params:
+      action: start
+      label: job_performance  
+  - Identifier: WaitForFinishedJobs
+    Method: WaitForFinishedJobs
+    Params:
+      action: start
+      labelSelector: group = test-job
+  - Identifier: JobLifecycleLatency
+    Method: JobLifecycleLatency
+    Params:
+      action: start
+      labelSelector: group = test-job
+  - Identifier: GenericPrometheusQuery
+    Method: GenericPrometheusQuery
+    Params:
+      action: start
+      metricName: API request latency
+      metricVersion: v1
+      unit: s
+      queries:
+        - name: total_jobs_scheduled
+          query: kueue_admitted_workloads_total{cluster_queue="{{$clusterQueue}}"} 
+        - name: total_pods_scheduled
+          query: count(kube_pod_info{namespace=~"{{$namespacePrefix}}.*"})
+        - name: avg_pod_running_time
+          query: (avg(kube_pod_completion_time{namespace=~"{{$namespacePrefix}}.*"} - kube_pod_start_time{namespace=~"{{$namespacePrefix}}.*"}))
+        - name: perc_90_pod_completion_time
+          query: quantile(0.90, kube_pod_completion_time{namespace=~"{{$namespacePrefix}}.*"} - kube_pod_start_time{namespace=~"{{$namespacePrefix}}.*"})
+        - name: avg_pod_waiting_time
+          query: (avg(kube_pod_start_time{namespace=~"{{$namespacePrefix}}.*"} - kube_pod_created{namespace=~"{{$namespacePrefix}}.*"})) 
+        - name: perc_90_pod_waiting_time
+          query: quantile(0.90, kube_pod_start_time{namespace=~"{{$namespacePrefix}}.*"} - kube_pod_created{namespace=~"{{$namespacePrefix}}.*"})                
+        - name: max_job_throughput
+          query: max_over_time(sum(rate(kueue_admitted_workloads_total{cluster_queue="{{$clusterQueue}}"}[1m]))[{{$testTimeout}}:5s])                               
+- name: Sleep
+  measurements:
+  - Identifier: sleep
+    Method: Sleep
+    Params:
+      duration: 10s            
+- name: Create local queue
+  phases: 
+  - namespaceRange:
+      min: 1
+      max: {{$namespaces}}
+    replicasPerNamespace: 1
+    tuningSet: UniformQPS
+    objectBundle:
+    - basename: local-queue
+      objectTemplatePath: "local-queue.yaml"
+      templateFillMap:
+        LocalQueue: {{$localQueue}}
```

Comment:
do you need to set this? isn't the name set from `basename`?

Reply:
Fixed: https://github.com/kubernetes-sigs/kueue/pull/462/commits/4c4ccc622a94448bfe1495281a96585cfc587111